client/app/src/main/java/com/example/criengine/Fragments/NotificationFragment.java

```diff
@@ -0,0 +1,69 @@
+package com.example.criengine.Fragments;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.Button;
+import android.widget.ListView;
+
+import com.example.criengine.Activities.AddBookActivity;
+import com.example.criengine.Adapters.NotificationAdapter;
+import com.example.criengine.Database.DatabaseWrapper;
+import com.example.criengine.Adapters.MyBooksAdapter;
+import com.example.criengine.Objects.Book;
+import com.example.criengine.Objects.Notification;
+import com.example.criengine.Objects.Profile;
+import com.example.criengine.R;
+import com.google.android.gms.tasks.OnSuccessListener;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Notification Fragment.
+ * Displays all current notifications for the user.
+ */
+public class NotificationFragment extends Fragment {
+    private NotificationAdapter notificationAdapter;
+    private ListView notificationListView;
+    private DatabaseWrapper dbw;
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.activity_notification, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Dummy Code starts here.
+        final Profile profile = new Profile();
```

Comment:
is this profile supposed to be the user profile? gotten from the login procedure or something?

Reply:
Yeah its the user profile

Reply:
shouldn't profile be created somewhere and passed in? dependency injection?

Reply:
although i'm not sure that's setup yet

Reply:
Profile is obtained during the login procedure but I have no idea how it will be passed in and Sam also said not to put anything related to notifications into the database. So for testing we have to make dummy code.